src/main/cpp/chassis/headingStates/MaintainHeading.cpp

```diff
@@ -27,6 +28,7 @@ using std::string;
 
 MaintainHeading::MaintainHeading() : ISwerveDriveOrientation(ChassisOptionEnums::HeadingOption::MAINTAIN)
 {
+    m_controller.EnableContinuousInput(-180.0, 180.0);
```

Comment:
Should this be -pi/2 to pi/2 because our inputs are radians?

Reply:
Probably- good catch I'll make change when we get there